tests/test_api/test_feeds.py

```diff
@@ -137,6 +137,45 @@ def test_post_subscription_validURL_old_feed(self, test_client, user, real_feed)
         check_valid_header_type(r.headers)
         assert r.status_code == 201
 
+    def test_post_subscription_invalid_feed_url_0(self, test_client, user):
```

Comment:
Since we just check for a BOZO (invalid) xml format, I think these will all be covered. Maybe just have one additional test that passes a valid URL, but invalid feed URL (i.e. http://time.com).